### DISCUSSION TITLE
I started timer from terminal but somehow ended with over 500K hour timer.

### DISCUSSION THREAD

**Discussion by psmyda** [SelfControlApp/selfcontrol#844] [Q&A]
So I was trying to incorporate self-control timer to my shortcut and used some existing shortcut but it bugged somehow.
<img width="431" alt="image" src="https://github.com/SelfControlApp/selfcontrol/assets/3226321/b3e9959c-de3c-472a-9128-ed3be27f575f">
I was able to get rid of block and helper is uninstalled but I can't make this timer to zero itself :D 

Any suggestions what files I still need to remove?

**Comment by cstigler** [comment]
Hi @psmyda, if you send me an email I can help you out - charlie+sc844@charliestigler.com
